.github/actions/gcloud/action.yml

```diff
@@ -0,0 +1,21 @@
+name: gcloud
+description: Configure gcloud
+inputs:
+  cloud_sdk_service_account_key:
+    required: true
+  gcp_project:
+    required: true
+  gcp_zone:
+    required: true
+  create:
+    required: false
+    description: Set to true to create a new cluster. The linkerd CLI should have been installed in the runner beforehand.
+  destroy:
+    required: false
+    description: Set to false to create a new cluster. The linkerd CLI should have been installed in the runner beforehand.
```

Comment:
should `create`/`delete` just be a single string input called `action`? feels like the api should not allow the user to set both of these bools to true or both to false. also consider documenting that if these are not set, the action's default behavior is just to configure the gcloud tooling.

Reply:
Ok, an enum for this makes more sense indeed :ok_hand: